src/main/configurations/Translate/Configuration_ZdsHeeft.xml

```diff
@@ -116,10 +116,22 @@
                 preserveInput="true"
                 xpathExpression="exists(/result/exception)"
                 >
-                <Forward name="then" path="EXCEPTION"/>
+                <Forward name="then" path="ResultWithErrorMessage"/>
                 <Forward name="else" path="CheckForStatusExists"/>
             </XmlIfPipe>
 
+            <XsltPipe
+                name="ResultWithErrorMessage"
+                styleSheetName="Common/xsl/BuildError.xsl"
+                >
+                <Param name="code" value="TechnicalError" /> <!-- codes: TechnicalError, TranslationError, ConfigurationError-->
+                <Param name="reason" pattern="ActualiseerStatus datumstatusgezet equal to existing status" />
+                <Param name="details" sessionKey="StatusBody" xpathExpression="concat(ZgwStatus/datumStatusGezet, ' is equal to existing status')"/>
+                <Param name="detailsXml" sessionKey="ZgwStatussen" type="DOMDOC" />
+                <Forward name="success" path="EXIT" />
```

Comment:
Those error handling pipes always need to exclusively exit to EXCEPTION. Otherwise an error is not actually flagged as an error anymore.